.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: go
+sudo: false
+go:
+    - 1.7
+install:
+    - cd cluster-state-service; make get-deps; cd ../
+    - cd daemon-scheduler; make get-deps; cd ../
+script:
+    - cd cluster-state-service; make; make release; cd ../
```

Comment:
`make` will run unit tests, but will also build the binary. And make release will build the docker image. Do we care about having Travis verify it can build the binary and docker image, or just run the unit tests?